deploy/helm/sumologic/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
I know that it's not required to use SemVer2 for `appVersion`, but I'm wondering how are we going to increment appVersion later?

Reply:
we can decide whether we want to use it since it's an optional field. if we do, it should be the same as the docker image stable release version

Reply:
changed to `0.0.0` for now